### ISSUE TITLE
prepare-commit-msg hook prints a warning

### ISSUE THREAD

**Issue by belgattitude** [CaptainHookPhp/captainhook#14]
Hey captainhook, 

Got the 'prepare-commit-msg' hook installed through the plugin but there's a php notice (undefined var)... This notice prevents me to commit anything (console exit -1)... I've just removed the hooks for now but would be nice to have a fix.

captainhooks rocks !

```
PHP Notice:  Undefined index: prepare-commit-msg in vendor/captainhook/captainhook/src/Console/Application/Hook.php on line 133
PHP Stack trace:
PHP   1. {main}() ....git/hooks/prepare-commit-msg:0
PHP   2. CaptainHook\App\Console\Application\Hook->run() .../.git/hooks/prepare-commit-msg:18
```

**Comment by sebastianfeldmann** [comment]
I’ll have a look at it after dinner and fix it ASAP

**Comment by sebastianfeldmann** [comment]
The issue should be fixed with version 3.0.2
The respective command class was missing. The issue is fixed but the `prepare-commit-message` hook is not working correctly.
I hope I can fix this with the next update

**Comment by belgattitude** [comment]
Thanks a lot ! Works like a charm.
